typescript/db/src/resolvers/workspace.ts

```diff
@@ -103,6 +110,43 @@ const createWorkspace = async (
     },
     user
   );
+  if (createdWorkspaceId) {
+    const now = new Date();
+    const workspaceSlug = getSlug(args.data.name);
```

Comment:
You should get the slug back from the repository instead of regenerating one (and potentially breaking the SRP).

Reply:
Fixed